python/docs/source/index.md

```diff
@@ -3,94 +3,119 @@ hide-toc: true
 html_theme.sidebar_secondary.remove: true
 title: Kaskada Timestreams
 ---
+# Introduction
 
 <div class="px-4 py-5 my-5 text-center">
     <img class="d-block mx-auto mb-4" src="_static/kaskada.svg" alt="" width="auto">
-    <h1 class="display-5 fw-bold">Event-processing for AI applications.</h1>
+    <h1 class="display-5 fw-bold">Real-Time AI without the fuss.</h1>
     <div class="col-lg-7 mx-auto">
-      <p class="lead mb-4">Next-generation, real-time and historic event processing.
+      <p class="lead mb-4">Kaskada is a simple streaming engine that connects AI models to real-time & historical data.
       </p>
     </div>
 </div>
 
+(what-it-provides)=
```

Comment:
Unless you're relying on these links, you shouldn't need the `(anchor-name)=` on headings. Sphinx will automatically generate an Anchor based on the heading for linking to (and in fact, this will add a second anchor, that won't show up automatically while scrolling). Probably only necessary if you want to link to it and be able to change the heading without having the link stop working (but sphinx will complain about internal links if the heading isn't there).

Reply:
Cool - wasn't sure if they were important